backend/src/org/akvo/lumen/auth.clj

```diff
@@ -33,13 +33,13 @@
 (defn wrap-jwt
   "Go get cert from Keycloak and feed it to wrap-jwt-claims. Keycloak url can
   be configured via the KEYCLOAK_URL env var."
-  [handler issuer]
+  [handler keycloak-url realm]
   (try
-    (let [certs (-> (str issuer "/realms/akvo/protocol/openid-connect/certs")
+    (let [issuer (str keycloak-url "/realms/" realm)
+          certs (-> (str issuer "/protocol/openid-connect/certs")
                     client/get
                     :body)]
       (jwt/wrap-jwt-claims handler (jwt/rsa-key certs 0) issuer))
     (catch Exception e
```

Comment:
Agree, we should probably kill the server. I added a try / catch to be able to print "Could not get cert from Keycloak" to make it easier to understand if you are new to the system.